runtime_config_linux.go

```diff
@@ -182,7 +182,9 @@ type CPU struct {
 
 // Pids for Linux cgroup 'pids' resource management (Linux 4.3)
 type Pids struct {
-	// Maximum number of PIDs. A value < 0 implies "no limit".
+	// Maximum number of PIDs. A value <= 0 implies "no limit".
+	// While a limit of 0 is technically valid, it is precisely identical to
+	// a limit of 1, so there is no distinction.
```

Comment:
Do you want both “≤ 0 means no limit” and “0 is the same as 1”?  I'd expect either “≤ 0 means no limit and 0 is the same as -1” or “< 0 means no limit and 0 is the same as 1”.  Or does 1 really mean “no limit”?